### PR TITLE
Add tests for remaining menu-button examples

### PR DESCRIPTION
**Pull request by spectranaut**
This adds tests for:
- menu-button/menu-button-links.html 
- menu-button/menu-button-actions-active-descendant.html

### CODE REVIEW

**Comment by spectranaut** [comment]
this is ready for review, @jessebeach @sh0ji or @tatermelon :)

File: test/tests/menu-button-actions-active-descendant.js [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+const exampleFile = 'menu-button/menu-button-actions-active-descendant.html';
+
+const ex = {
+  menubuttonSelector: '#ex1 button',
+  menuSelector: '#ex1 [role="menu"]',
+  menuitemSelector: '#ex1 [role="menuitem"]',
+  numMenuitems: 4,
+  lastactionSelector: '#action_output',
+  defaultAriaActivedescentantVal: 'mi1'
```
Comment (jessebeach) [inline test/tests/menu-button-actions-active-descendant.js]: 'descendant'

File: test/tests/menu-button-actions-active-descendant.js [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+};
+
+const checkFocus = function (t, selector, index) {
+  return t.context.session.executeScript(function () {
+    const [selector, index] = arguments;
+    let items = document.querySelectorAll(selector);
```
Comment (jessebeach) [inline test/tests/menu-button-actions-active-descendant.js]: `const`

File: test/tests/menu-button-links.js [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+  menubuttonSelector: '#ex1 button',
+  menuSelector: '#ex1 [role="menu"]',
+  menuitemSelector: '#ex1 [role="menuitem"]',
+  numMenuitems: 6
+};
+
+const checkFocus = function (t, selector, index) {
+  return t.context.session.executeScript(function () {
+    const [selector, index] = arguments;
+    let items = document.querySelectorAll(selector);
```
Comment (jessebeach) [inline test/tests/menu-button-links.js]: `const`

File: test/util/assertAriaActivedescendant.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+
+/**
+ * Assert the aria-activedescendant focus is correctly set to the item that has
+ * at the appropriate index.
+ *
+ * @param {obj} t                  - ava execution object
+ * @param {String} ariaDescendantSelector - selector for element with aria-activeDescendant set
```
Comment (jessebeach) [inline test/util/assertAriaActivedescendant.js]: nit: "activedescendant". The casing is always lower-case on these attributes.

File: test/util/assertAriaActivedescendant.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+ * @param {String} optionsSelector - selector to select list of canidate elements for focus
+ * @param {Number} index           - index of element in list returned by optionsSelector with focus
+ */
+module.exports = async function assertAriaSelectedAndActivedescendant (t, ariaDescendantSelector, optionsSelector, index) {
+
+  // Confrirm aria-activedescendant refers to the correct option
+
+  let options = await t.context.session
```
Comment (jessebeach) [inline test/util/assertAriaActivedescendant.js]: `const` and `const`.

File: test/util/assertAriaActivedescendant.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      .getAttribute('aria-activedescendant'),
+    optionId,
+    'aria-activedescendant should be set to ' + optionId + ' for items: ' + ariaDescendantSelector
+  );
+
+  // Confirm the focus is on the aria-activedescendent element
+
+  let focused = await t.context.session.executeScript(function () {
+    const selector = arguments[0];
+    let item = document.querySelector(selector);
```
Comment (jessebeach) [inline test/util/assertAriaActivedescendant.js]: `const`.

**Pull request merged**